internal/tfimport/importer/google_resource_manager_lien.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package importer
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/template"
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/terraform"
+)
+
+// ResourceManagerLien defines a struct with the necessary information for a google_resource_manager_lien to be imported.
+type ResourceManagerLien struct{}
+
+// ImportID returns the ID of the resource to use in importing.
+func (i *ResourceManagerLien) ImportID(rc terraform.ResourceChange, pcv ConfigMap, interactive bool) (string, error) {
+	// Get required fields.
+	fieldsMap, err := loadFields([]string{"parent", "name"}, interactive, rc.Change.After, pcv)
```

Comment:
The name is not known from beforehand from what I remember. 

`name - A system-generated unique identifier for this Lien.`

So how would loadFields know the value?

In V2 I used to list all liens and match against one with the same set of restrictions.

Reply:
I originally had it like this because loadFields will ask the user for it. I assumed the user would look it up themselves.

I've now modified the PR to fetch the liens remotely and present them as choices.